app/models/piece.rb

```diff
@@ -18,6 +18,27 @@ def move!(x,y)
     end
   end
 
+  #Inital Capture Logic:
+  #Need game model to test id of game
+
+  #Set initial captured status to false
+  def default_captured_status
+    self.captured = false if self.captured = nil
+  end
```

Comment:
Test coverage is also missing for this method, which I recommend adding as well.